examples/llm/vdb_upload/pipeline.py

```diff
@@ -11,107 +11,90 @@
 # limitations under the License.
+
 import logging
 import time
+import typing
+
+from vdb_upload.helper import process_vdb_sources
 
 from morpheus.config import Config
-from morpheus.config import PipelineModes
-from morpheus.pipeline.linear_pipeline import LinearPipeline
+from morpheus.pipeline.pipeline import Pipeline
 from morpheus.stages.general.monitor_stage import MonitorStage
 from morpheus.stages.general.trigger_stage import TriggerStage
 from morpheus.stages.inference.triton_inference_stage import TritonInferenceStage
-from morpheus.stages.input.rss_source_stage import RSSSourceStage
 from morpheus.stages.output.write_to_vector_db_stage import WriteToVectorDBStage
-from morpheus.stages.preprocess.deserialize_stage import DeserializeStage
 from morpheus.stages.preprocess.preprocess_nlp_stage import PreprocessNLPStage
 
-from ..common.utils import build_milvus_config
-from ..common.utils import build_rss_urls
-from ..common.web_scraper_stage import WebScraperStage
-
 logger = logging.getLogger(__name__)
 
 
-def pipeline(num_threads: int,
-             pipeline_batch_size: int,
-             model_max_batch_size: int,
-             model_fea_length: int,
-             embedding_size: int,
-             model_name: str,
-             isolate_embeddings: bool,
-             stop_after: int,
-             enable_cache: bool,
-             interval_secs: int,
-             run_indefinitely: bool,
-             vector_db_uri: str,
-             vector_db_service: str,
-             vector_db_resource_name: str,
-             triton_server_url: str):
+def pipeline(pipeline_config: Config,
+             source_config: typing.List,
+             vdb_config: typing.Dict,
+             embeddings_config: typing.Dict,
+             tokenizer_config: typing.Dict) -> float:
+    """
+    Sets up and runs a data processing pipeline based on provided configurations.
 
-    config = Config()
-    config.mode = PipelineModes.NLP
+    Parameters
+    ----------
+    source_config : Dict
+        Configuration for data sources (e.g., 'rss', 'filesystem').
+    vdb_config : Dict
+        Configuration for the vector database.
+    pipeline_config : Dict
+        General configuration for the pipeline (e.g., number of threads, batch sizes).
+    embeddings_config : Dict
+        Configuration for embeddings (e.g., model name, embedding size).
 
-    # Below properties are specified by the command line
-    config.num_threads = num_threads
-    config.pipeline_batch_size = pipeline_batch_size
-    config.model_max_batch_size = model_max_batch_size
-    config.feature_length = model_fea_length
-    config.edge_buffer_size = 128
+    Returns
+    -------
+    float
+        The start time of the pipeline execution.
+    """
 
-    config.class_labels = [str(i) for i in range(embedding_size)]
+    isolate_embeddings = embeddings_config.get('isolate_embeddings', False)
 
-    pipe = LinearPipeline(config)
+    pipeline_config.debug = True
```

Comment:
Maybe only set this only if the log level is set to debug?

Reply:
Good catch, I think this was a testing thing.